CONTRIBUTING.md

```diff
@@ -101,6 +101,60 @@ get going:
   your SSH key passphrase to the managing agent, so that you don't have to
   keep authenticating to GitHub. You need to do this again after every reboot.
 
+## Supplementary Developer Contribution Guidelines:
+1. Ensuring a Single Commit  
+To ensure that your pull request contains only a single commit, you may need to squash your commits.

```

Comment:
Actually I think community doesn't restrict a PR to have only one commit. We recommend that developers can squash some similar commits based on the content.

Reply:
@Shelley-BaoYue Thanks for your explanation, I have modified this part.